CHANGELOG.md

```diff
@@ -7,10 +7,17 @@ All notable changes to this project will be documented in this file.
 ### Changed
 ### Fixed
 
+## [2.17.40] - 2023-03-20
+### Added
+
+ - Added session timeout warning modal.  After 25 minutes of inactivity a
```

Comment:
The text we show users says 30 minutes, this says 25 :D

Reply:
30 minutes is the total session time.  25 mins of inactivity + 5 mins of warning modal = 30 mins